koku/api/report/aws/query_handler.py

```diff
@@ -83,8 +83,11 @@ def annotations(self):
             annotations['usage_units'] = Coalesce(self._mapper.usage_units_key, Value(units_fallback))
         # { query_param: database_field_name }
         fields = self._mapper.provider_map.get('annotations')
+        prefix_removed_parameters_list = map(lambda x: x if ':' not in x else x.split(':', maxsplit=1)[1],
```

Comment:
Consider a list comprehension here. Not required, but I find them them easier to read than map, lambda. 

```
prefix_removed_params_list = [param_key.split(':', maxsplit=1)[1] if ':' in param_key else param_key for param_key in parameters.get('group_by', {}).keys()]
```